libs/client/ui/pages/src/lib/register/register.tsx

```diff
@@ -205,7 +206,11 @@ export const Register: React.FC<RegisterProps> = inject(
                 valid={acceptTosValid}
                 label={
                   <IonNote mode="md">
-                    Ich akzeptiere Datenschutz und Nutzungsbedingungen
+                    Ich akzeptiere die{' '}
+                    <IonRouterLink routerLink={`/privacy-police`}>
+                      Datenschutzerklärung
+                    </IonRouterLink>{' '}
+                    und Nutzungsbedingungen
```

Comment:
werden mir Nutzungsbedingungen no kriga? sunsch evtl. ussalöscha

Reply:
Stimmt i lösches lieber noch ussa sus isches verwirrend. Falls do noch was kut, füg is widda hinzu